docs/core/tools/dotnet-install-script.md

```diff
@@ -151,9 +151,15 @@ The install scripts do not update the registry on Windows. They just download th
   - `aspnetcore` - the `Microsoft.AspNetCore.App` shared runtime.
   - `windowsdesktop` - the `Microsoft.WindowsDesktop.App` shared runtime.
 
-- **`--runtime-id <RID>`**
+- **`--runtime-id <RID>` [Deprecated]**
 
-  Specifies the [runtime identifier](../rid-catalog.md) for which the tools are being installed. Use `linux-x64` for portable Linux. (Only valid for Linux/macOS.)
+  Specifies the [runtime identifier](../rid-catalog.md) for which the tools are being installed. Use `linux-x64` for portable Linux. (Only valid for Linux/macOS and for versions earlier than .Net Core 2.1.)
+
+  **`--os <OPERATING_SYSTEM>`**
+
+  Specifies the operating system for which the tools are being installed (Valid for .Net Core 2.1 and above.). Possible values are: `osx`, `linux`, `linux-musl`, `freebsd`, `rhel.6`.
+
+  Specifying this option should only be necessary when running the script from an uncommon Unix based operating system and the script fails to determine for which of the supported operating systems to install the tools.
```

Comment:
I would consider also another reason -  overriding the automatic OS detection for any other reason, for example installing linux package on rhel.6, etc.  Alternative generic wording can be: `the parameter is optional and should only be used when it is required to override the operating system detected by a script`.